### PR TITLE
Add `BadgeList` component

### PR DESCRIPTION
**Pull request by rusnacok**

### CODE REVIEW

File: catalog/src/main/java/kiwi/orbit/compose/catalog/screens/MainScreen.kt [hunk 1]
```diff
@@ -76,6 +76,7 @@ internal fun MainScreen(
     val controls = listOf(
         MenuItem("Alert", Icons.Alert) { Destinations.Alert },
         MenuItem("Badge", Icons.Deals) { Destinations.Badge },
+        MenuItem("BadgeList", Icons.AirplaneReturn) { Destinations.BadgeList },
```
Comment (hrach) [inline catalog/src/main/java/kiwi/orbit/compose/catalog/screens/MainScreen.kt:79]: Please use Icons.List

File: catalog/src/main/java/kiwi/orbit/compose/catalog/screens/BadgeListScreen.kt [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                Text(text = "This is simple BadgeList item")
+            }
+        }
+
+        Text(
+            text = "Badge List Normal Minor",
+            style = OrbitTheme.typography.title4,
+        )
+
+        BadgeListSmall(contentEmphasis = ContentEmphasis.Minor) {
```
Comment (hrach) [inline catalog/src/main/java/kiwi/orbit/compose/catalog/screens/BadgeListScreen.kt:131]: BadgeList only

**Review by hrach — CHANGES_REQUESTED** [review]
LGTM! Two little things to fix.

File: catalog/src/main/java/kiwi/orbit/compose/catalog/screens/MainScreen.kt (hunk 1 above)
Comment (rusnacok) [reply catalog/src/main/java/kiwi/orbit/compose/catalog/screens/MainScreen.kt:79]: OK

**Review by hrach — APPROVED** [review]

**Comment by hrach** [comment]
👍

**Pull request merged**